dapp/src/utils/numbers.ts

```diff
@@ -218,3 +219,11 @@ export const addLeadingZero = (num: number): string =>
 
 export const getPercentValue = (value: number, maxValue: number) =>
   (value * 100) / maxValue
+
+export const getNumberWithSign = (number: number) =>
+  new Intl.NumberFormat("en-US", {
+    signDisplay: "exceptZero",
+  }).format(number)
```

Comment:
We can hoist it up to not create the instance on every `getNumberWithSign` function call.